src/nbl/video/IOpenGL_LogicalDevice.h

```diff
@@ -1017,6 +1037,13 @@ class IOpenGL_LogicalDevice : public ILogicalDevice, protected impl::IOpenGL_Log
     virtual void destroySync(GLsync sync) = 0;
     virtual void setObjectDebugName(GLenum id, GLuint object, GLsizei len, const GLchar* label) = 0;
     virtual void destroyQueryPool(COpenGLQueryPool* qp) = 0;
+    virtual int getEGLAPI() = 0;
+    virtual EGLConfig getEglConfig() = 0;
+    virtual EGLContext getEglContext() = 0;
+    virtual const COpenGLFeatureMap* getGlFeatures() = 0;
+    virtual std::pair<EGLint, EGLint> getGlVersion() = 0;
```

Comment:
you need any of these?

Reply:
all of these get used, previously they were just part of the createSwapchain function inside IOpenGL_LogicalDevice.h